mff/MFF.py

```diff
@@ -70,15 +93,26 @@ class MFF:
     def __init__(self,
                  df: pd.DataFrame,
                  forecaster = DefaultForecaster(),
```

Comment:
Consider adding a type hint here:

```suggestion
                 forecaster: BaseForecaster = DefaultForecaster(),
```

Kindly note that also you'll need to have the relevant import statement in this file:

```
from sktime.forecasting.base import BaseForecaster
```